tools/src/launcher/launcher-config.yaml

```diff
@@ -0,0 +1,76 @@
+# Path to the node executable
+nodePath: .stack-work/install/x86_64-linux-nopie/lts-9.1/8.0.2/bin/cardano-node
```

Comment:
This is a bad idea, since it doesn't work with Nix.

Reply:
This is just an example. It should be clear how to modify this under different circumstances.